src/Microsoft.Extensions.DependencyInjection/exceptions.netcore.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "OldTypeId": "public static class Microsoft.Extensions.DependencyInjection.ServiceCollectionContainerBuilderExtensions",
+    "OldMemberId": "public static System.IServiceProvider BuildServiceProvider(this Microsoft.Extensions.DependencyInjection.IServiceCollection services)",
+    "NewTypeId": "public static class Microsoft.Extensions.DependencyInjection.ServiceCollectionContainerBuilderExtensions",
+    "NewMemberId": "public static Microsoft.Extensions.DependencyInjection.ServiceProvider BuildServiceProvider(this Microsoft.Extensions.DependencyInjection.IServiceCollection services)",
+    "Kind": "Modification"
```

Comment:
FYI API Check doesn't actually care much about additions to regular classes and it has a bug when handling `Modification` exceptions in `interface`s (aspnet/BuildTools#154). Might as well make this a `Removal` and leave out the `New*` properties.